src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -425,6 +425,13 @@ export class AttachToDropdown extends SelectBox {
 		if ((connProviderIds && connProviderIds.length === 0) || currentKernel === noKernel) {
 			this.setOptions([msgLocalHost]);
 		} else {
+			// Removes kernel to connection provider mapping of SQL kernel to KUSTO connection
```

Comment:
Shouldn't this logic be more generic than just for Kusto? The idea is anything can contribute kernel alias' right?